lib/membrane_h264_plugin/parser/decoder_configuration_record.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Membrane.H264.Parser.DecoderConfigurationRecord do
+  @moduledoc """
+  Utility functions for parsing AVC Configuration Record
+  """
```

Comment:
I'm not sure if those docs should be generated with `mix docs`. If so, add docs for `t()` and `parse`